app/models/epds/cayuse_project.rb

```diff
@@ -23,15 +23,21 @@ class Epds::CayuseProject < Epds::Connection
   self.primary_key = "PROJECT_ID"
 
   has_many :cayuse_awards, foreign_key: 'PROJECT_ID'
-  has_many :cayuse_research_teams, -> { where(ROLE: 'Lead Principal Investigator') }, through: :cayuse_awards 
+  has_many :cayuse_research_teams, through: :cayuse_awards #-> { where(ROLE: 'Lead Principal Investigator') },
```

Comment:
I believe this `where` statement is still needed.  We don't want all of the users.  Just the Lead Principal Investigators.  If that list is more than 1, show the `Multiple Principal Investigators` message.  If you remove the where you will be getting all types of people.